tensorflow/java/src/gen/cc/ops/op_generator.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
The annotation processor won't be part of the final jar, it will only be invoked while building it, as a Bazel `java_plugin`, and won't be accessible publicly after. The Ops file that it generates, though, will be included in the library.

So the need of keeping CC and Java generators separated is still present but we could drop off the /ops subfolder as you suggested?